LINK/usr/bin/miq_postgres_upgrade.sh

```diff
@@ -99,9 +104,41 @@ mount -a
 # init the new cluster
 su - postgres -c "initdb -D ${NEW_PGSQL_DIR}/data-new"
 
+sed -i.bak -e "s/^#*shared_preload_libraries.*/shared_preload_libraries = 'pglogical'/" ${NEW_PGSQL_DIR}/data-new/postgresql.conf
```

Comment:
interesting...I do like the idea of uncommenting and editing though, so if someone searches the file there aren't two instances (which if they don't see the second, they could uncomment the first, and create problems)